React/Base/RCTMultipartStreamReader.m

```diff
@@ -76,7 +76,11 @@ - (void)emitProgress:(NSDictionary *)headers
   // Throttle progress events so we don't send more that around 60 per second.
   CFTimeInterval currentTime = CACurrentMediaTime();
 
+<<<<<<< HEAD
   NSInteger headersContentLength = headers[@"Content-Length"] != nil ? [headers[@"Content-Length"] integerValue] : 0;
+=======
```

Comment:
Please back out the changes to this file -- it's not related to your new feature and I think the issue with this file has already been fixed separately by someone else.